[docs] Add Quick Start page (#441)
ostis-ai/sc-machine: README.md

[![CI](https://github.com/ostis-ai/sc-machine/actions/workflows/main.yml/badge.svg)](https://github.com/ostis-ai/sc-machine/actions/workflows/main.yml)
[![codecov](https://codecov.io/gh/ostis-ai/sc-machine/branch/main/graph/badge.svg?token=WU8O9Z1DNL)](https://codecov.io/gh/ostis-ai/sc-machine)
[![license](https://img.shields.io/badge/License-MIT-yellow.svg)](COPYING.MIT)
[![docker](https://img.shields.io/docker/v/ostis/sc-machine?arch=amd64&label=Docker&logo=Docker&sort=semver)](https://hub.docker.com/r/ostis/sc-machine)

[![Quick Start](https://img.shields.io/badge/-Quick%20Start-black?style=for-the-badge&logo=rocket)](https://ostis-ai.github.io/sc-machine/quick_start)
[![Docs](https://img.shields.io/badge/Docs-gray?style=for-the-badge&logo=read-the-docs)](https://ostis-ai.github.io/sc-machine)
[![Community](https://img.shields.io/badge/-Community-teal?style=for-the-badge&logo=matrix)](https://app.element.io/index.html#/room/#ostis_tech_support:matrix.org)

Short version: **sc-machine** is a software package that emulates **semantic computer behavior** by storing and processing knowledge in a semantic network. It uses **agent-based approach** to process knowledge graphs.

> or, if you'll look at it from an angle of databases...
> 
> sc-machine is a **platform-independent graph database management system** that can store / retrieve knowledge graphs and run tasks (agents) on them.

Reviewer: It looks like coquetry, I suggest a more formal look
```suggestion
From a graph database perspective sc-machine is a **platform-independent graph database management system** that can store / retrieve knowledge graphs and run tasks (agents) on them.
```

> sc-machine is a **platform-independent graph database management system** that can store / retrieve knowledge graphs and run tasks (agents) on them.

Reviewer: Why is it platform-independent?

sc-machine stores and processes knowledge graphs represented in the SC-code (language of the universal knowledge representation). Theoretical basis of sc-machine is provided by the [**OSTIS Technology**](https://github.com/ostis-ai).

> Both declarative (data, data structures, documentation, tasks specification, etc.) and procedural
> (programs, modules, systems, communication between systems) knowledge is represented using that same language.

Reviewer: ```suggestion
Both declarative (data, data structures, documentation, problems specification, etc.) and procedural
(agents, programs, modules, systems, communication between systems) knowledge is represented using that same language.
```

  - `sc-server` - a Websocket server that provides a network API to communicate with storage using JSON;
  - `sc-builder` loads information from SCs files into the storage (SCs-code is one of the external representation languages of the SC-code).

## Documentation

- A brief user manual and developer docs are hosted on our [GitHub Pages](https://ostis-ai.github.io/sc-machine).
  - <details>
      <summary>Build documentation locally</summary>

>     ```sh

Reviewer 2: There are some bugs with the display of some characters in markdown text on earlier versions of python.

>     ```sh

Author: I am not sure what "python12" is supposed to mean, do you refer to Python 3.12?

>     ```sh

Reviewer 2: Oh, yeah)

    pip3 install mkdocs mkdocs-material
    mkdocs serve
    # and open http://127.0.0.1:8000/ in your browser
    ```
    </details>
- Full documentation, including:

  - core concepts
  - rationale behind the sc-machine
  - system design
  - software interfaces

>   is redistributed in a form of the SCn-TeX document [compiled in PDF format](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).

Reviewer: I'm not sure this is an alternative. One documentation has something that the other one doesn't and vice versa. I suggest referring to this documentation as containing more rigorous theoretical calculations
